wifiphisher.py

```diff
@@ -403,7 +398,6 @@ def dhcp_conf(interface):
 
 def dhcp(dhcpconf, mon_iface):
     os.system('echo > /var/lib/misc/dnsmasq.leases')
-    dhcp = Popen(['dnsmasq', '-C', dhcpconf], stdout=PIPE, stderr=DN)
```

Comment:
This actually starts dnsmasq responsible for the DHCP / DNS settings. You may remove the variable assignment if you want to.